core/services/commander/main.py

```diff
@@ -204,7 +206,25 @@ async def root() -> Any:
     return HTMLResponse(content=html_content, status_code=200)
 
 
+def setup_ssh() -> None:
+    try:
+        # check if id_rsa.pub exists, creates a new one if it doesnt
+        if not os.path.exists("/home/pi/.ssh/id_rsa.pub"):
+            subprocess.run(shlex.split("ssh-keygen -t rsa -f /home/pi/.ssh/id_rsa -q -N ''"), check=True)
+        # add id_rsa.pub to authorized_keys if not there already
+        if not os.path.exists("/home/pi/.ssh/authorized_keys"):
+            subprocess.run(shlex.split("touch /home/pi/.ssh/authorized_keys"), check=True)
+        with open("/home/pi/.ssh/authorized_keys", "r+", encoding="utf-8") as f:
+            if not any("ssh-rsa" in line for line in f):
+                with open("/home/pi/.ssh/id_rsa.pub", encoding="utf-8") as key_file:
+                    f.write(key_file.read())
+    except Exception as error:
```

Comment:
Will this work if there is already a ssh-rsa key configured by the user ?

Reply:
yep, I deleted it and created one in the pi. it still worked fine